pkg/daemon/ceph/agent/flexvolume/util.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2019 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package flexvolume
+
+import (
+	"os/exec"
+	"strings"
+)
+
+// GetRookFlexBinaryPath returns the path of rook flex volume driver
+func GetRookFlexBinaryPath() string {
+	path, err := exec.LookPath(flexvolumeDriverFileName)
+	if err != nil {
+		return usrBinDir
+	}
+	return strings.Replace(path, flexvolumeDriverFileName, "", 1)
```

Comment:
If you import the "path" package, you could call `path.Dir(p)` to remove the filename from the end.

Reply:
cool will do it

Reply:
done